crates/cubecl-core/src/frontend/element/cube_elem.rs

```diff
@@ -50,3 +50,21 @@ impl From<UInt> for ExpandElement {
         ))
     }
 }
+
+impl From<AtomicI32> for ExpandElement {
+    fn from(_value: AtomicI32) -> Self {
+        todo!()
+    }
+}
+
+impl From<AtomicI64> for ExpandElement {
+    fn from(_value: AtomicI64) -> Self {
+        todo!()
+    }
+}
+
+impl From<AtomicUInt> for ExpandElement {
+    fn from(_value: AtomicUInt) -> Self {
+        todo!()
+    }
+}
```

Comment:
If not necessary I would remove those. Probably linked to the generic `From` parameter I guess.